pipeline/mutate/mutator_hydrator.go

```diff
@@ -77,22 +82,56 @@ type externalAPIConfig struct {
 	Retry *retryConfig `json:"retry"`
 }
 
+type cacheConfig struct {
+	Ttl string `json:"ttl"`
+}
+
 type MutatorHydratorConfig struct {
-	Api externalAPIConfig `json:"api"`
+	Api   externalAPIConfig `json:"api"`
+	Cache cacheConfig       `json:"cache"`
 }
 
 type mutatorHydratorDependencies interface {
 	x.RegistryLogger
 }
 
 func NewMutatorHydrator(c configuration.Provider, d mutatorHydratorDependencies) *MutatorHydrator {
-	return &MutatorHydrator{c: c, d: d, client: httpx.NewResilientClientLatencyToleranceSmall(nil)}
+	cache, _ := ristretto.NewCache(&ristretto.Config{
+		NumCounters: 10000,
+		MaxCost:     1 << 25,
+		BufferItems: 64,
```

Comment:
I think it makes sense to annotate the config a bit:

```suggestion
		// This will hold about 1000 unique mutation responses.
		NumCounters: 10000,
		// Allocate a max of 32MB
		MaxCost:     1 << 25,
		// This is a best-practice value.
		BufferItems: 64,
```